Components/Cart/cart.html

```diff
@@ -13,70 +13,78 @@
 
 <body>
     <div class="navigation-container">
-        <div class="logo-input">
+        <div class="">
             <div class="nav-header">
-                <h1>Daily Fashion</h1>
-            </div>
-            <div class="search-input-container"><input class="header-input" type="search">
-                <img class=" input-icon" src="https://img.icons8.com/material/28/000000/search--v1.png" />
+                <a class="site-title" href="/Components/Homepage/homepage.html">
+                    <h1>DAILY FASHION</h1>
```

Comment:
Since we are using react, we should use className  insted of class

Reply:
No No... this is a Static project with HTML & CSS only 🙂